controllers/rating_controller.go

```diff
@@ -88,11 +88,38 @@ func (r *RatingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return reconcile.Result{Requeue: true}, err
 	}
 
-	if err := r.CreatePipelineRun(logger, ctx, &instance); err != nil {
-		logger.Error(err, "")
+	instanceDeepCopy := instance.DeepCopy()
+	component := corev1alpha1.Component{}
+	if err := r.Client.Get(ctx, types.NamespacedName{Namespace: instance.Namespace, Name: instance.Spec.ComponentName}, &component); err != nil {
 		return reconcile.Result{}, err
 	}
 
+	repository := corev1alpha1.Repository{}
+	if err := r.Client.Get(ctx, types.NamespacedName{Namespace: instance.Namespace, Name: component.Status.RepositoryRef.Name}, &repository); err != nil {
+		return reconcile.Result{}, err
+	}
+	if repository.Spec.EnableRating {
+		if err := r.CreatePipelineRun(logger, ctx, &instance); err != nil {
+			logger.Error(err, "")
+			return reconcile.Result{}, err
+		}
+	} else if instanceDeepCopy.Status.ConditionedStatus.Conditions[0].Status == "False" {
```

Comment:
else if instanceDeepCopy.Status.ConditionedStatus.Conditions[0].Status != "False" {
instanceDeepCopy.Status.ConditionedStatus = corev1alpha1.ConditionedStatus{
			Conditions: []corev1alpha1.Condition{
				{
					Status:             v1.ConditionFalse,
					LastTransitionTime: metav1.Now(),
					Reason:             corev1alpha1.ReasonReconcileError,
					Message:            "Rating disabled in component's repository",
					Type:               corev1alpha1.TypeReady,
				},
			},
		}
		err := r.Client.Status().Patch(ctx, instanceDeepCopy, client.MergeFrom(&instance))
		return reconcile.Result{}, err
}

	return ctrl.Result{}, nil